app/components/multiple_checked_select_component.rb

```diff
@@ -1,7 +1,9 @@
 # frozen_string_literal: true
 
 class MultipleCheckedSelectComponent < ViewComponent::Base
-  def initialize(name:, options:, selected:)
+  # @param id [String] uniquely identifies the MultipleCheckedSelect (mcs) field. '_mcs_field' will be appended
+  def initialize(id:, name:, options:, selected:)
+    @id = "#{id}_mcs_field"
```

Comment:
adding id to the mcs field, it would make it easier to test and identify each field uniquely in case we use multiple mcs on a page

Reply:
If we use multiple fields, they should have different labels.